### ISSUE TITLE
Make configs not reset when a plugin fails to load.

### ISSUE THREAD

**Issue by PintTheDragon** [Exiled-Team/EXILED#784]
Currently, if a plugin fails to load, its config will be reset/deleted. This is fine if you remove it but if a breaking change happens, your plugin stops working and its config reset for everyone.

**Comment by NaoUnderscore** [comment]
This is pretty annoying indeed.

**Comment by PintTheDragon** [comment]
> I have to agree with @PintTheDragon, it's really annoying if you setup a good config and then misspelled something or did something wrong and the config just resetting then.

Oh no, I'm not talking about that. It's totally fine for a field in a plugin's config to reset if it was entered invalidly, that's the fault of whoever's configuring it. I'm talking about if a plugin fails to load for whatever reason, its configs disappear. So if a game update happens, a plugin could just lose all of its configs for no reason, even though those configs are valid.

**Comment by Marco15453** [comment]
Yeah, i was writing it wrong in the comment i mean the resetting for a field in a plugin's config is fine.
